aes/Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "aes"
-version = "0.6.0"
-description = "Facade for AES (Rijndael) block ciphers implementations"
+version = "0.7.0-pre"
```

Comment:
I bumped the version worried this might potentially be a breaking change, but I think it may not be?

Reply:
Changed it back to v0.6.0